### PR TITLE
Move loading the installable components to confirm page

### PR DESCRIPTION
**Pull request by dgolovin**
Fix moves loading the list of components available for installation
to confirmation controller in preparation to loading configuration
from remote location.

### CODE REVIEW

**Comment by dgolovin** [comment]
Test this

**Comment by dgolovin** [comment]
test this

**Comment by dgolovin** [comment]
test this

**Comment by jrichter1** [comment]
test this

File: test/unit/pages/confirm/controller-test.js [hunk 1]
```diff
@@ -45,9 +52,14 @@ describe('ConfirmController', function() {
...
-    for (var installer of installerDataSvc.allInstallables().values()) {
-      sandbox.stub(installer, 'detectExistingInstall').resolves();
-    }
+
+    sandbox.stub(CDKInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(CygwinInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(HypervInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(JdkInstall.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(VirtualBoxInstallWindows.prototype, 'detectExistingInstall').resolves();
+    sandbox.stub(VirtualBoxInstallDarwin.prototype, 'detectExistingInstall').resolves();
```
Comment (jrichter1) [inline test/unit/pages/confirm/controller-test.js]: I think it should be possible to do this with the original loop (which really looks nicer), it just needs to be moved below the confirmController instantiation.

Comment (dgolovin) [inline test/unit/pages/confirm/controller-test.js]: That sounds good, I was just testing if it is possible to move components loading to confirmation page, so I can at some point implement updating requirements.json from remote location.

**Pull request merged**